site/_data/authorsData.json

```diff
@@ -647,5 +647,10 @@
     "country": "US",
     "twitter": "bgalbs",
     "image": "image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg"
+  },
+  "twifkak": {
+    "twitter": "twifkak",
+    "github": "twifkak",
+    "image": "image/rULxC7pPw3PFS4o9xr7v8isFmCv1/fDaDWpuKLKDm4mjnt0jV.jpg"
```

Comment:
```suggestion
    "image": "image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/cshsSdVAseOneqg2APon.jpg"
```

Reply:
Per the [images and video guidelines](https://web.dev/handbook/markup-media/), author avatars should be 384x384.

Reply:
As per a separate conversation, I think we're good using a higher resolution image URL, and allowing the image CDN to downsize it as needed.

Reply:
+1. @twifkak, feel free to reject this change, as my interpretation of the images and video guidelines may have been too strict.

Reply:
OK, will reject. Yeah, I uploaded the high resolution per the recommendation atop the [Choose a file](https://web.dev/handbook/markup-media/#choose-a-file) section of that doc:

> "Upload a high quality image... Our image CDN... will resize the image so you don't have to."